cli/internal/plugins/bzr/bzr.go

```diff
@@ -12,41 +12,70 @@ import (
 	"github.com/gobuffalo/buffalo-cli/plugins/plugprint"
 )
 
-type Buffalo struct{}
+const (
+	name        = "bzr"
+	description = "Provides bzr related hooks to Buffalo applications."
+)
+
+//Ensuring bzr is a describer
+var _ plugprint.Describer = Buffalo{}
 
+//Ensuring bzr is a buffalo Plugin
+var _ plugins.Plugin = Buffalo{}
+
+//Ensuring bzr is a buffalo buildcmd.Versioner
 var _ buildcmd.Versioner = &Buffalo{}
 
-func (b *Buffalo) BuildVersion(ctx context.Context, root string) (string, error) {
-	if _, err := exec.LookPath("bzr"); err != nil {
-		return "", err
-	}
+//testConfig is used to mock command things when testing.
+var testConfig = struct {
```

Comment:
again, no package state, and there should be any mention of testing outside of the tests themselves.